app/v1/models/request/amendDividends/AmendDividendsRequestBody.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.request.amendDividends
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{JsPath, OWrites, Reads}
+import utils.JsonUtils
+
+case class AmendDividendsRequestBody(foreignDividend: Option[Seq[AmendForeignDividendItem]],
+                                     dividendIncomeReceivedWhilstAbroad: Option[Seq[AmendDividendIncomeReceivedWhilstAbroadItem]],
+                                     stockDividend: Option[AmendStockDividend],
+                                     redeemableShares: Option[AmendRedeemableShares],
+                                     bonusIssuesOfSecurities: Option[AmendBonusIssuesOfSecurities],
+                                     closeCompanyLoansWrittenOff: Option[AmendCloseCompanyLoansWrittenOff],
+                                    )
```

Comment:
how does this compile 
```suggestion
                                     closeCompanyLoansWrittenOff: Option[AmendCloseCompanyLoansWrittenOff])
```